src/main/java/me/xmrvizzy/skyblocker/config/SkyblockerConfig.java

```diff
@@ -530,10 +539,18 @@ public String toString() {
     }
 
     /**
-     * Registers the config to AutoConfig and register commands to open the config screen.
+     * Registers the config to AutoConfig and registers commands to open the config screen.
      */
     public static void init() {
-        AutoConfig.register(SkyblockerConfig.class, GsonConfigSerializer::new);
+        Gson gson = new GsonBuilder()
+                .setPrettyPrinting()
+                .registerTypeHierarchyAdapter(Text.class, new Text.Serializer())
+                .registerTypeHierarchyAdapter(net.minecraft.text.Style.class, new net.minecraft.text.Style.Serializer())
```

Comment:
Can you import `Style`.

Reply:
I wasn't able to import `Style` because the enum in the class had the same name which conflicted. I changed the name of that enum so that we could import `Style` there.